homeworks/nagynorbert/3_conditionals_loops/ticket_booking.py

```diff
@@ -0,0 +1,95 @@
+# Extra homework - Nagy Norbert
+from pprint import pprint
+
+films = ["alien", "it", "amelie csodas elete", "hallo, hallo", "csengetett mylord"]
+
+# define title of film
+while True:
```

Comment:
Ez is lehetne egy function, amit az elején meghívsz és visszaadja a választott film címét.